pkg/eks/components/NodeGroup.vue

```diff
@@ -640,7 +640,10 @@ export default defineComponent({
         </KeyValue>
       </div>
     </div>
-    <hr class="mb-20">
+    <hr
+      class="mb-20"
+      role="none"
```

Comment:
Does `role="separator"` fit better than `none`?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/separator_role

Reply:
If it's purely to address the issues on the our report, then it's much easier to "not read" them. I found at least a couple of issues on the minor's in the report in different areas. I am sure that for some the `role="separator"` would  work just fine, but I did not look at all those rendered HR's in our UI. Just too many hits to go one by one.